File: TiaCodegen/Commands/Comparisons/Ge.cs
using TiaCodegen.Interfaces;

namespace TiaCodegen.Commands.Comparisons
{
    public class Ge : CompareOperator // >=
    {
        public Ge(params IOperationOrSignal[] operationOrSignals)
            : base(operationOrSignals)
        { }
    }
}

File: TiaCodegen-ts/src/Commands/Comparisons/Ge.ts
import { IOperationOrSignal } from '../../Interfaces/IOperationOrSignal.js';
import { CompareOperator } from './CompareOperator.js';

export class Ge extends CompareOperator {
    constructor(...operationOrSignals: IOperationOrSignal[]) {
        super(...operationOrSignals);
    }
}
